frontend/frontend.go

```diff
@@ -11,14 +11,14 @@ import (
 // after the response has been delivered to the client.
 type TrackerLogic interface {
 	// HandleAnnounce generates a response for an Announce.
```

Comment:
Returns the updated context, the generated AnnounceResponse and no error on success; nil and error on failure.